hw/application_fpga/fw/tk1/main.c

```diff
@@ -115,18 +115,20 @@ static void compute_cdi(const uint8_t digest[32], const uint8_t use_uss,
 {
 	uint32_t local_cdi[8];
 	blake2s_ctx secure_ctx;
+	uint32_t rnd;
```

Comment:
Nits: Use a better name than just rnd. rnd_sleep for example.

Reply:
Ok.